lib/class-wp-rest-block-editor-settings-controller.php

```diff
@@ -16,7 +16,7 @@ class WP_REST_Block_Editor_Settings_Controller extends WP_REST_Controller {
 	 * Constructs the controller.
 	 */
 	public function __construct() {
-		$this->namespace = '__experimental/wp-block-editor/v1';
+		$this->namespace = 'wp-block-editor/v1';
```

Comment:
I've checked some stable endpoints in the gutenberg and wordpress-develop codebase and this is what I see: `wp/v2/pattern-directory`, `wp/v2/widget-types`, `wp/v2/widgets`, `wp/v2/block-directory`, `wp/v2/block-types`, `wp/v2/comments`.

Following the convention of existing endpoints, I wonder if this should be `wp/v2/block-editor-settings` instead, `wp/v2` as namespace and `block-editor-settings` as `rest_base`.

I'd welcome thoughts from people more experienced with the REST API naming schema, cc @TimothyBJacobs

Reply:
Thanks for the suggestions @nosolosw!

From [this comment](https://github.com/WordPress/gutenberg/pull/29969#issuecomment-802137951) it was suggested to use this naming instead of following the `wp/v2/` convention, although it'd make things easier if it was using the same structure as the existing ones 😅

Reply:
Yeah the intention to use the different namespace is that `wp/v2` is for modeling data, whereas this is purely a block editor API returning configuration values.

There really is no plan to introduce a `wp/v3` for numerous reasons, but putting this in a block editor namespace would allow for more freedom in introducing newer versions for breaking changes.

This would match what we did for Site Health in having a `wp-site-health/v1` namespace. I think the only inconsistency here would be that ideally the `block-directory` and `pattern-directory` endpoints would've been part of this namespace as well.

Reply:
I see, thanks for the context. Let's do this then.

As a data point for the future, I'd like to provide the thought that it's difficult to reason about why/when something is data or a setting from the perspective of a consumer of the API. By looking at the [reference documentation](https://developer.wordpress.org/rest-api/reference/), I see endpoints that are arguably config: post types, block types, site settings to name a few.

Reply:
Sounds good! Let's keep the `wp-site-health/v1` namespace 👍 

Can we then proceed with removing the `__experimental` part? Thanks!